exist-versioning-release.md

```diff
@@ -0,0 +1,315 @@
+
+    **NOTE:** The timeframe between a RC and final release (or next RC) must be short, and therefore is **limited to two weeks**. During this time **ONLY patch bugfix** PRs should be merged.
+
+6. Checkout the `eXist-3.2.0-RC1` tag and create product builds for publication:
+    ```
+    $ git checkout eXist-3.2.0-RC1
+    $ ./build.sh jnlp-unsign-all all jnlp-sign-exist jnlp-sign-core
+    $ ./build.sh installer installer-exe app dist-war
+    ```
+
+## Motivation
+
+The goal of this proposal is to ensure the theory and practice for each new version of eXist is solid is well understood by core developers and the eXist community. Versioning practices adopted during the development of eXist 3.0 and some well-intentioned mistakes in its public release led the authors to propose a new scheme which could avoid these problems and establish a firmer foundation in versioning and release practices.
+
+### The Old Way
+
+During the development of eXist 3.0, the version-related properties in `$EXIST_HOME/build.proprties` diverged and looked like this:
+
+```
+project.version = 3.0.RC2
+project.version.numeric = 3.0.3
+```
+
+Here there are two different version numbers above: `project.version` and `project.version.numeric`. The second version number was introduced in an attempt to assist many users who were running custom-compiled versions and needing to detect API changes during the very extended release candidate phase. The divergence in version numbers caused real confusion and consternation among users who tried to communicate these version numbers with each other. 
+
+When eXist 3.0 was released, these properties were manually modified for sake of expediency and therefore did not match the Git tag `eXist-3.0`. Rather, its `$EXIST_HOME/build.properties` contained the following version components:
+
+```
+project.version = 3.0.0
+project.version.numeric = 3.0.4
+```
+
+Our goal is to prevent such a divergence in versioning from creeping back into eXist and to ensure a clean versioning system to serve eXist through its future development and release cycles.
+
+### The New Way
+
+Once this proposal is adopted, eXist will have a single version number, forumlated according to the precise principles of Semantic Versioning, captured in a single property:
+
+```
+project.version = 3.0.0
+```
+
+Once a stable release has been tagged, we will immediately initiate the next version, assuming a `MINOR` release, unless the core developers select a `PATCH` or `MAJOR` version; this next version will have a `LABEL` appended, e.g., `3.1.0-SNAPSHOT`, which will persist until 3.1.0 is released, unless a new `PATCH` or `MAJOR` version must be released first.
+
+We can call the installer (and/or packages) anything we want, but
+it would be sensible for them to reflect the version number clearly.
+So for simplicity we suggest just using the same version as is in
+project.version, i.e.:
```

Comment:
Uggh. An artifact of copying and pasting Markdown in and out of GoogleDocs :-/
Just fixed it.